### ISSUE TITLE
Put back assert and use names wihtout tildes for artificial tables

### ISSUE THREAD

**Issue by samwebster** [samwebster/times-excel-reader#37]
See https://github.com/samwebster/times-excel-reader/pull/28#discussion_r1118093699

**Comment by olejandro** [comment]
@siddharth-krishna should we put the assert back already?

**Comment by olejandro** [comment]
Or will this be irrelevant once we transition away from `times_mapping.txt`?

**Comment by siddharth-krishna** [comment]
Yes, perhaps once we move to the `TimesModelData` class in #11 this is no longer an issue?
